### ISSUE TITLE
Classification table not using attribute?

### ISSUE THREAD

**Issue by smalers** [OpenWaterFoundation/owf-app-infomapper-ng#68]
I'm looking the categorized symbol for water districts.  The classification table has `` for the `value` column.  Instead, it should have unique values corresponding to the `classificationAttribute` property.  For example, `classificationAttribute=DISTRICT` and the `value` column should have 64 or 7, etc.

The example as is has `classificationAttribute=NAME`, which is OK but a bit more cumbersome because  `value` would need to be `South Platte - Balzac to Stateline`, etc. 

Basically the `value` column contents don't make sense.
